lib/stat_tracker.rb

```diff
@@ -67,6 +67,13 @@ def percentage_home_wins
         ((win_count.to_f / @game_stats_data.length)).round(2)
     end
 
+    def percentage_visitor_wins
```

Comment:
Logic looks solid, good use of reusing code structure.
Good to know that .to_f only needs to be called on the numerator or denominator to be effective.